src/openinide.js

```diff
@@ -206,7 +206,12 @@
                 fileString = foundRootPath.replace(/\/$/, "") + "/" + fileString.replace(/^\//,"");
                 bHasRootPath = true;
             }
-            
+
+            // at least on Meteor projects it was not working starting with /
+            if (fileString.startsWith('/')) {
+                fileString = fileString.substring(1);
+            }
+
```

Comment:
Instead of adding this block, please remove the `|| (urlParse.path[0] == '/')` part on line 135. That will also result in removal of the leading slash.

Reply:
Yes, i know that the issue is not only concerning Meteor. I think we can merge this PR, i'd just like to see an example Meteor URL to be sure I'm not missing something.